src/extension.ts

```diff
@@ -3,14 +3,31 @@ import * as _ from 'lodash';
 import * as path from 'path'
 import * as fs from 'fs';
 
+/**
+ * Attempts to require the gullconfig.json file and
+ * returns it to the main function for futher use.
+ *
+ */
+async function readConfig() {
```

Comment:
I don't see anything asynchronous happening in this function.  "require" is synchronous.

Reply:
Nice catch! That was from what I had prior to switching to just doing require